dashboard/app/controllers/projects_controller.rb

```diff
@@ -66,7 +66,9 @@ def remix
     if STANDALONE_PROJECTS[params[:key].to_sym][:login_required]
       authenticate_user!
     end
-    new_channel_id = create_channel(nil, params[:channel_id])
```

Comment:
confused. why don't I see `create_channel` in https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/projects_controller.rb ?